tests/test_snmpuser.rb

```diff
@@ -59,11 +59,11 @@ def teardown
     assert_empty(delta, 'Users not deleted after test!')
   end
 
```

Comment:
fyi, we've done some cleanups in the test files this week that may cause some merge conflicts for you. It's mostly just method name changes to make them shorter:

https://github.com/cisco/cisco-network-node-utils/commit/b91c16854a612a2dc5d522a6a7cda6dbb0ef5dd6

This was done simply to make the test results more readable in the regression reports.